apps/omg_watcher_info/lib/omg_watcher_info/supervisor.ex

```diff
@@ -64,19 +64,19 @@ defmodule OMG.WatcherInfo.Supervisor do
       {OMG.WatcherInfo.BlockApplicationConsumer, []},
       {OMG.WatcherInfo.DepositConsumer, []},
       Supervisor.child_spec(
-        {OMG.WatcherInfo.ExitConsumer, [topic: {:root_chain, "ExitStarted"}]},
+        {OMG.WatcherInfo.ExitConsumer, [topic: {:root_chain, "ExitStarted"}, event_type: :standard_exit]},
         id: :std_exit_consumer
       ),
       Supervisor.child_spec(
-        {OMG.WatcherInfo.ExitConsumer, [topic: {:watcher, "InFlightExitStarted"}]},
+        {OMG.WatcherInfo.ExitConsumer, [topic: {:watcher, "InFlightExitStarted"}, event_type: :in_flight_exit]},
         id: :ife_exit_consumer
       ),
       Supervisor.child_spec(
-        {OMG.WatcherInfo.ExitConsumer, [topic: {:watcher, "InFlightTxOutputPiggybacked"}]},
+        {OMG.WatcherInfo.ExitConsumer, [topic: {:watcher, "InFlightTxOutputPiggybacked"}, event_type: :in_flight_exit]},
         id: :ife_exit_output_piggybacked_consumer
       ),
       Supervisor.child_spec(
-        {OMG.WatcherInfo.ExitConsumer, [topic: {:watcher, "InFlightExitOutputWithdrawn"}]},
```

Comment:
Could we name the last two events differently? Say `in_flight_exit_output_piggybacked` and `in_flight_exit_output_withdrawn`

From the consuming applications' perspective (mobile wallet, block explorer, web wallet) we would want to distinguish between these events in future features - i.e. tracking the progression of an exit.

So concerned that storing every event related to IFEs under the same `event_type` will cause headaches down the line

Reply:
Simple answer - we can - this PR makes it possible (I tune the UTXO query which supports 2 event types, so we can have more without effort).

I'm reluctant to do this in this PR because we not need it and we aren't clear about the future requirements. I'd prefer to see the product requirement and consider the changes in the system that satisfy it. 

One specific objection I have is as you might know from the PR which introduced these events `output_withdrawn` is a last resort in case syncing went weird, so normally I expect to see former two event types in the live system in majority of cases.

Reply:
I missed a lot of contexts here to provide a useful comment. But as long as we leave the possibility open in the future for client application to come to an accurate state of the UTXO based on the events reported by the watcher. I think its 🆗